src/SelfService/Application/CapabilityApplicationService.cs

```diff
@@ -310,4 +313,11 @@ public async Task<bool> DoesOnlyModifyRequiredProperties(string jsonMetadata, Ca
 
         return true;
     }
+
+    [TransactionalBoundary]
```

Comment:
Do we need a TransactionalBoundary for read operations?
It's not hurting, but I thought this was mostly for write?

Reply:
was a relic from a copy/paste (whoops)